### ISSUE TITLE
Make table displays more consistent wrt contest

### ISSUE THREAD

**Issue by thijskh** [DOMjudge/domjudge#599]
I think it's desirable to make our jury table views more consistent between eachother in how they deal with multiple contests. There are currently three flavours (more or less): show all data (with a contest column in the table), idem but only for contests currently 'active', and the third way is to only show information about the contest that is selected in the top-right menu.

This is the current situation:

Page | All contests | Active contests | Selected Contest
------------ | ------------- | ------------- | -------------
audit log | :heavy_check_mark:  | |
balloons | :heavy_check_mark:  | | :white_check_mark: 
clarifications | | :heavy_check_mark:  |  :white_check_mark: 
contests | :heavy_check_mark:  | |
internal errors | :heavy_check_mark:  | |
problems | :heavy_check_mark:  | |
users | :heavy_check_mark:  | |
rejudgings | :heavy_check_mark:  | |
refresh cache  | | | :heavy_check_mark:
scoreboard  | | | :heavy_check_mark:
statistics  | | | :heavy_check_mark:
submissions  | | | :heavy_check_mark:
teams (+cat,affil) | :heavy_check_mark:  | |

Tables not currently contest-bound: executables, judgehosts, judgehost restrictions, languages

The current situation is probably not very consistent because many tables are carried over from the pre-multi-contest times and a per case change was made.

**Comment by thijskh** [comment]
I've added  :white_check_mark:  as an indication of my proposed changes.

Rows in tables like auditlog, teams, rejudgings, internal errors, problems can have a contest association but do not necessarily have one or exactly one. I think there the most confusion is avoided by always showing all, with a contest column where appropriate.

We know the system with the selected contest works well because we use it for submissions and scoreboard since the beginning. So it makes sense to me to make clarifications and balloons behave the same as submissions.

For clarifications we should consider what to do with the count in the menu bar. Only current contest or all unanswered in active contests?

**Comment by eldering** [comment]
I agree with making balloons and clarifications show only the selected contest.

Regarding what to do with the unanswered clarifications count, I don't reallly know. I think it's bad UI design to make it show the count for all (active) contests, and then when you click on it only see the clarifications of the selected contest, which thus may not match. Also I don't really know how people use DOMjudge when running multiple active contests.

**Comment by thijskh** [comment]
I have changed this for Balloons in b9139d81df7045d59ab0c4ee04ad02698f1cf101. I think it makes sense.

I've reconsidered clarifications and also the concerns above. I'm not sure it would be wrong to change it but also not whether it would be right. I think I lack understanding also of how people would want to work with clarifications in multiple parallel running contests. In any case the current way of working has not led to ((m)any) complaints that I know of. So I'm for now just leaving it as it is. We can always change it if we have some more concrete motivation for it.
